### ISSUE TITLE
[Bug] => Python 3 treats TabErrors as syntax errors

### ISSUE THREAD

**Issue by cclauss** [theInfectedDrake/TIDoS-Framework#41]
Indentation should be all tabs or all spaces but not a mix of the two in Python 3.

https://travis-ci.com/theInfectedDrake/TIDoS-Framework/jobs/150198095#L477

**Comment by theInfectedDrake** [comment]
Now I guess you understand the reason the reason why I defaulted Travis to build on Python 2 only for present. Check the updated [.travis.yml](https://github.com/theInfectedDrake/TIDoS-Framework/blob/master/.travis.yml)

**Comment by cclauss** [comment]
451 days until Python 2 end of life...  http://pythonclock.org

**Comment by cclauss** [comment]
A complete rewrite is not at all necessary.  Small cleanup != rewrite.

#37 was the bulk of the work and that is already merged.  TabErrors are just whitespace changes.

**Comment by theInfectedDrake** [comment]
Nah, I am not just doing the whole rewrite for this petty indentation error, the stuff is have planned for v2 has provisions that there are bulk additions that have to be made to almost all modules. #37 was the first PR of this repo and was huge, I seriously didn't want to spill your work away just like that. 

To be honest, I wrote this code in a worse+spaghetti way `check the source`, very casually cause I never expected TIDoS to grow this big. I mean like look at the way I wrote the code 😂, its not even a real framework structure. But since it has grown to this, a total code refactor is highly necessary. 🙂

**Comment by cclauss** [comment]
Are you married to tabs or could we run 
https://github.com/python/cpython/blob/master/Tools/scripts/reindent.py on the codebase?

**Comment by theInfectedDrake** [comment]
On the way to be fixed 😀👌.
